ee/be/modules/350-node-local-dns/monitoring/prometheus-rules/cache-plugin.yaml

```diff
@@ -0,0 +1,37 @@
+- name: d8.node-local-dns.cache
+  rules:
+    - alert: D8NodeLocalDNSCacheSuccessEvictionsHigh
+      expr: |
+        increase(coredns_cache_evictions_total{type="success", job="node-local-dns"}[5m]) > 100
+      for: 5m
+      labels:
+        severity_level: "6"
+        tier: cluster
+      annotations:
+        plk_markup_format: markdown
+        plk_protocol_version: "1"
+        summary: node-local-dns success cache is being frequently evicted.
+        description: |
+          The `success` cache in node-local-dns is evicting entries frequently (>100 in 5 minutes).
+
+          This could mean cache capacity is too small (e.g. `success 39936`), or traffic patterns changed.
```

Comment:
```suggestion
          This could mean there are too many different DNS records application have to resolve (to many Services?). So, node-local-dns cache capacity is too small (`success 39936` is currently configured), or traffic patterns changed.
```